scripts/build-index.js

```diff
@@ -223,13 +223,13 @@ function main() {
     process.stderr.write("Writing data/attack.json…\n");
     fs.writeFileSync("data/attack.json", JSON.stringify(attackOidLookup));
 
-    process.stderr.write("Writing data/lunr-index.json…\n");
+    process.stderr.write("Writing data/lunr-index.jsonx…\n");
```

Comment:
You and I know why the file has a `.jsonx` extension, but future collaborators on this project will be pretty confused. Add a comment above this that explains that mozilla blocks addons with large .`.json` files and this is a workaround.